SimCalorimetry/CaloSimAlgos/interface/CaloHitResponse.h

```diff
@@ -35,9 +35,17 @@ class CaloHitResponse {
   typedef std::map<DetId, CaloSamples> AnalogSignalMap;
   // get this from somewhere external
   enum { BUNCHSPACE = 25 };
-
-  CaloHitResponse(const CaloVSimParameterMap *parameterMap, const CaloVShape *shape);
-  CaloHitResponse(const CaloVSimParameterMap *parameterMap, const CaloShapes *shapes);
+  const double dt = 0.5;
```

Comment:
```suggestion
  constexpr double dt = 0.5;
  constexpr int invdt = 2;
```

Reply:
This leads to `error: non-static data member 'dt' declared 'constexpr'`

Shall I use `static constexpr` instead?

Reply:
I'm wondering why not just 
static const (int/double) ?

Reply:
I would use` static constexpr `if so...
But honestly I did not try it: just do whatever does not give errors

Reply:
Ok, that seems to work fine